after/syntax/rst.vim

```diff
@@ -0,0 +1,26 @@
+if dracula#should_abort('rst')
+    finish
+endif
+
+hi! link rstComment                             Comment
+hi! link rstTransition                          Comment
+hi! link rstCodeBlock                           DraculaGreen
+hi! link rstInlineLiteral                       DraculaGreen
+hi! link rstLiteralBlock                        DraculaGreen
+hi! link rstQuotedLiteralBlock                  DraculaGreen
+hi! link rstStandaloneHyperlink                 DraculaLink
+hi! link rstStrongEmphasis                      DraculaOrangeBold
+hi! link rstSections                            DraculaPurpleBold
+hi! link rstEmphasis                            DraculaYellowItalic
+hi! link rstDirective                           Keyword
+hi! link rstSubstitutionDefinition              Keyword
```

Comment:
I was not sure where to put RST Directive and Substitution Definition. I did consider `Label`, `Macro`, `Typedef` and `Define`. At the end, I choose `Keyword` because how significant Directive and Substitution are in RST.